docs/examples/data_transfer_objects/factory/paginated_return_data.py

```diff
@@ -1,13 +1,15 @@
 from datetime import datetime
 
-from sqlalchemy.orm import Mapped
+from sqlalchemy.orm import DeclarativeBase, Mapped
 
 from litestar import Litestar, get
+from litestar.contrib.sqlalchemy.base import CommonTableAttributes, UUIDPrimaryKey
 from litestar.contrib.sqlalchemy.dto import SQLAlchemyDTO
 from litestar.dto import DTOConfig
 from litestar.pagination import ClassicPagination
 
-from .my_lib import Base
+
+class Base(CommonTableAttributes, UUIDPrimaryKey, DeclarativeBase): ...
```

Comment:
if you are using stuff from advanced alchemy, you might as well directly use `UUIDBase` and make things simpler